Simulator robot simplify (#148)
srobo/docs: competition-simulator/programming/index.md

### Motors

Your robot has one motor board attached, the left wheel in port `m0`, and the right wheel in `m1`.

> The motor boards do not have part codes, and so need to be indexed using `0`.

Reviewer: ```suggestion
The motor board does not have a part code, so it needs to be indexed using `0`.
```
